### DISCUSSION TITLE
Online OWL workshop - live assembly and demonstration

### DISCUSSION THREAD

**Discussion by geezacoleman** [geezacoleman/OpenWeedLocator#29] [Announcements]
The OpenWeedLocator (OWL) live assembly and demonstration will guide you through the entire process of building a device, installing software before demonstrating a working unit towards the end of the 90 minute workshop. It will cover:

1. The purpose of the OWL project and OWL capability
2. Walk through of the parts list
3. Step-by-step hardware assembly
4. Step-by-step software installation (quick method)
5. Demonstration of working unit
6. Future plans for the OWL

**Date/time:** Wednesday 4th May, 12-1:30pm AEST
**Register:** [Eventbrite](https://www.eventbrite.com/e/openweedlocator-live-assembly-and-demonstration-tickets-305069871197) 

Looking forward to seeing you there!

**Comment by sunlite71** [comment]
I'll be appreciated if there is a downloadable record video for the above workshop :-)

**Reply by geezacoleman** [reply]
Sure! I probably should have added this before. While the discussion of software is fairly short, just note that some settings have changed since this video was made. I think it's probably time to run another live session/recording for OWL assembly/running.

The full recording is available below:
[OpenWeedLocator: Guided Assembly and Demonstration](https://youtu.be/vZqNKogzz8k)

**Reply by sunlite71** [reply]
Dear geezacoleman, Thanks so much for your prompt reply! 
Just curious have you tested Owl with Esp32 CAM?

Your earliest reply will be highly appreciated :-)

thanks
